examples/runtime/main.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/performancecopilot/speed"
+)
+
+const (
+	// refresh interval
+	interval = time.Millisecond
+
+	// number of times metrics are updated
+	iter = 1000
+)
+
+// list of memory-related metric domains
+var memMetricInstances = []string{
+	"Alloc",
+	"TotalAlloc",
+	"Sys",
+	"Lookups",
+	"Mallocs",
+	"Frees",
+	"HeapAlloc",
+	"HeapSys",
+	"HeapIdle",
+	"HeapInuse",
+	"HeapReleased",
+	"HeapObjects",
+	"StackInuse",
+	"StackSys",
+	"MSpanInuse",
+	"MSpanSys",
+	"MCacheInuse",
+	"MCacheSys",
+	"BuckHashSys",
+	"GCSys",
+	"OtherSys",
+	"NextGC",
+	"LastGC",
+	"PauseTotalNs",
+	"PauseNs",
+	"PauseEnd",
+	"NumGC",
+}
+
+func main() {
+	cpuIndom, err := speed.NewPCPInstanceDomain(
+		"CPU Metrics",
+		[]string{"CGoCalls", "Goroutines"},
+	)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Please add a message here, like `log.Fatal("could not create InstanceDomain, error:", err)`